Formula/a/aws-nuke.rb

```diff
@@ -20,29 +20,16 @@ class AwsNuke < Formula
   depends_on "go" => :build
 
   def install
-    build_xdst="github.com/ekristen/aws-nuke/v#{version.major}/cmd"
-    ldflags = %W[
-      -s -w
-      -X #{build_xdst}.BuildVersion=#{version}
```

Comment:
Does
```
aws-nuke --version
```
still produce the right output after dropping this?

Reply:
```
> aws-nuke --version
aws-nuke version 3.0.0-dev
```
It seems that the version is hard-coded.

Reply:
OK, similaly as before, I think I can add the version manually.
It is now return the correct one.

```
> aws-nuke --version
aws-nuke version 3.22.0
```